spec.bs

```diff
@@ -3359,6 +3410,34 @@ algorithms to achieve the outcomes described in the above explanatory content.
   1. Return "`Allowed`."
 </div>
 
+<div algorithm>
+  <dfn>Should navigation response to navigation request be blocked by sandboxing flags?</dfn>
+
+  Given a [=navigation params=] (|navigationParams|), this algorithm returns "`Blocked`" or
+  "`Allowed`":
+
+  1. Let |navigable| be |navigationParams|'s [=navigation params/navigable=].
+
+  1. If |navigable| is not a [=fenced navigable container/fenced navigable=], then return
+     "`Allowed`".
+
+  1. Let |effectiveSandboxingFlags| be the |navigable|'s [=navigable/active browsing context=]'s
+     [=browsing context/fenced frame config instance=]'s [=fenced frame config instance/effective
+     sandboxing flags=].
```

Comment:
This concerns me a little bit, since I think we're pulling the _live_ list of flags here, right?

Could this set of sandboxing flags have changed from what it was observed while loading, to now? Should we be reaching for the snapshot of them at all?

Reply:
The fenced frame config effective sandboxing flags should stay the same. The only place it will be set is when a config-generating API sets it when first creating the config. It's probably better to not be touching the live navigable for this though, so I'll change it to the `sourceSnapshotParams`'s `target fenced frame config`'s `effective sandboxing flags`.

Reply:
Of if the `config` attribute was assigned mid-way through navigation, and we end up referencing the "new" config that's not associated with _this_ navigation anymore (although "this" navigation should be cancelled by some of the bail out points anyways, but there are not many of them). Anyways, I think the change you made is good.